website/docs/guides/internals.md

```diff
@@ -3,39 +3,37 @@ title: Internals
 sidebar_position: 45
 ---
 
-Even though Scala CLI exposes a simple interface to users, quite a number of steps happen
-under the hood when compiling or running even a single source file.
+Even though Scala CLI exposes a simple interface to users, quite a number of steps happen when compiling or running even a single source file.
+This page describes what happens under the hood when you run a Scala CLI command.
 
 
 ### Bloop
 
-Scala CLI uses Bloop to compile code. That way, it doesn't interface directly with scalac / dotty,
-and newly released Scala versions work out-of-the-box - no need to update Scala CLI itself.
+Scala CLI uses Bloop to compile code.
+That way, it doesn't interface directly with `scalac`, and newly released Scala versions work out of the box: there's no need to update Scala CLI itself.
 
-In more detail: Scala CLI connects to Bloop on the local machine via the default Bloop port (`8212`).
-If no Bloop instance listens on this port, Scala CLI fetches Bloop (via coursier), and starts it.
+The way this works is that Scala CLI connects to Bloop on the local machine at the default Bloop port, `8212`.
+If no Bloop instance is found on this port, Scala CLI fetches Bloop (via Coursier), and starts it.
 
-Right now, Scala CLI connects to Bloop via an open TCP port on your local machine, although this may change in
-the future for security reasons (see [`scalacenter/bloop#1562`](https://github.com/scalacenter/bloop/pull/1562)).
+:::note
+Right now, Scala CLI connects to Bloop via an open TCP port on your local machine, but this may change in the future for security reasons (see [`scalacenter/bloop#1562`](https://github.com/scalacenter/bloop/pull/1562)).
+:::
```

Comment:
@alexarchambault this is outdated, isn't it? We are using named sockets in most cases, aren't we?